### ISSUE TITLE
As a user I want to view stock level with colours on each product

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#100]

**Comment by lcabrini** [comment]
This story is too opinionated about how the function is supposed to look like. The user stories should only be about the functionality, not about how to present it.

**Comment by BorderITS** [comment]
is this ok

**Comment by lcabrini** [comment]
Yes, that is much better as it focuses only on what needs to be done, not how to present it.

**Comment by lcabrini** [comment]
But now we need to think about the role. Are all user supposed to view stock levels? Because a user is basically the lowest user above a visitor (a non-authenticated user).

**Comment by BorderITS** [comment]
ok then, what of the rest

**Comment by lcabrini** [comment]
The story itself is much better as I mentioned in my earlier comment. My only remaining question is which role should be required to be able to view stock levels?

**Comment by BorderITS** [comment]
warehouse manager but can't  sales person have this report on their dashboard ?

**Comment by lcabrini** [comment]
Yes, but I don't know how to go about it. Please refer to #110 

You can help out by doing some research of your own and contributing to that issue. Once we have a solution, this and any other story that seem to affect multiple roles can then be updated.
